tslint.json

```diff
@@ -76,63 +65,13 @@
       true,
       "as-needed"
     ],
+    "object-literal-sort-keys": false,
+    "ordered-imports": false,
     "quotemark": [
       true,
       "single"
     ],
-    "semicolon": {
```

Comment:
Why have we deleted these the tslint rules?

Reply:
using exactly the same lint rules as the other web app for consistency